R/summarize_weighted_buildout.R

```diff
@@ -0,0 +1,154 @@
+#' Summarize build-out based on the weight of each loan per sector per year
+#'
+#' Summarize build-out based on the weight of each loan per sector per year.
```

Comment:
Maybe remove one copy. The period "." will be wrong either in the title or the description, but that's fine. Maybe somthing to suggest to roxygen2.

Reply:
It'd be nice to read what "buildout" means.

Reply:
Agreed, I'm actually not set on the term "buildout". I'm hoping that as I write out the article/ methodology I can think of a better word

Reply:
It looks you settled for "percent-change", which I find more evocative than buildout.